### DISCUSSION TITLE
Disabling the lockip feature

### DISCUSSION THREAD

**Discussion by mrstirlic** [Zrips/CMI#6606] [Q&A]
How can I disable the lockip function in the CMI plugin, I don’t need this function, and even more so, it bothers me

**Comment by CloudeLecaw** [comment]
Simply dont use it and dont give permission for it.
Cmi has many Features that many not used by everyone..so simply dont use it, and if u dont want the /lockip then set it to false on alias.yml

**Comment by mrstirlic** [comment]
Alas, not in this case, even if I turn it off, it still won’t help me, I want to get rid of this function directly, because it does not allow me to enter the server from a different IP (if I logged in earlier under a different IP)

**Comment by CloudeLecaw** [comment]
Ah so you enabled it and wanne disable it now i see

Well first you should try to clear the list.

lockip (playername) [add/remove/list/clear] [ip]
